tools/tests/standard-minification.js

```diff
@@ -1,39 +0,0 @@
-import selftest, {Sandbox} from '../tool-testing/selftest.js';
```

Comment:
Quick note regarding why I've removed this test (and its supporting test app). The need to split stylesheets up with more than 4095 selectors was required to support IE 9. All modern (and supported) browsers can handle stylesheets with more than 4095 selectors, so I don't think there's much value in retaining this test (and its associated functionality). Let me know if anyone disagrees!